docs/standard/tour.md

```diff
@@ -91,7 +91,7 @@ In .NET, delegates are commonly used in event handlers, in defining asynchronous
 
 Generics allow the programmer to introduce a *type parameter* when designing their classes that allows the client code (the users of the type) to specify the exact type to use in place of the type parameter.
 
-Generics were added to help programmers implement generic data structures. Before their arrival in order for a type such as the `List` type to be generic, it would have to work with elements that were of type `object`. This had various performance and semantic problems, along with possible subtle runtime errors. The most notorious of the latter is when a data structure contains, for instance, both integers and strings, and an `InvalidCastException` is thrown on working with the list's members.
+Generics were added to help programmers implement generic data structures. Before their arrival, in order for a type such as the `List` type to be generic, it would have to work with elements that were of type `object`. This had various performance and semantic problems, along with possible subtle runtime errors. The most notorious of the latter is when a data structure contains, for instance, both integers and strings, and an <xref:System:InvalidCastException> is thrown on working with the list's members.
```

Comment:
```suggestion
Generics were added to help programmers implement generic data structures. Before their arrival, in order for a type such as the `List` type to be generic, it would have to work with elements that were of type `object`. This had various performance and semantic problems, along with possible subtle run-time errors. A common run-time error is when a data structure contains, for example, both integers and strings, and an <xref:System:InvalidCastException> is thrown while processing the list's members.
```

Reply:
@gewarren, Applied the suggestion. I feel like the "and" before "an InvalidCastException" is grammatically not correct and can be removed. May you confirm ? I'm not a native speaker so I might be mistaken.